### ISSUE TITLE
Adding Search Bar Support to Firefox

### ISSUE THREAD

**Issue by adityatelange** [microsoft/aka#1]
Hello Team,

We can add search bar support using a page which has `opensearch` config set. 

Contents of html:

```html
<link rel="search" type="application/opensearchdescription+xml" title="aka.ms" href="/opensearch.xml">
```

Contents of `opensearch.xml`

```xml
<?xml version="1.0" encoding="UTF-8" ?>
<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/" xmlns:moz="http://www.mozilla.org/2006/browser/search/">
  <ShortName>aka.ms</ShortName>
  <Description>Search aka.ms</Description>
  <InputEncoding>UTF-8</InputEncoding>
  <Image width="16" height="16" type="image/x-icon">https://www.microsoft.com/favicon.ico?v2</Image>
  <Url type="text/html" method="get" template="https://aka.ms/{searchTerms}"></Url>
</OpenSearchDescription>
```

### Demo: https://aka-ms.vercel.app (open in Firefox)

![image](https://user-images.githubusercontent.com/21258296/194717794-f31062f2-479e-4901-8e1d-d5ce5292fcd0.png)

![image](https://user-images.githubusercontent.com/21258296/194717851-caa77610-f88a-4780-94df-4dcabdd14a84.png)

**Comment by merill** [comment]
This looks good. Would you be able to submit this as a PR? Thanks!

**Comment by adityatelange** [comment]
I think a solution is already merged with PR #4. That solution is way more easier than mine. Thank you for consideration, closing.
